modules/gathering-data-specific-features.adoc

```diff
@@ -78,6 +78,9 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift4/ose-csi-driver-shared-resource-mustgather-rhel8:v<installed_version_secret_store>`
 |Data collection for the {secrets-store-operator}.
 
+|`registry.redhat.io/lvms4/lvms-must-gather-rhel9:v<installed-version-LVMS>`
```

Comment:
```suggestion
|`registry.redhat.io/lvms4/lvms-must-gather-rhel9:v<installed_version_LVMS>`
```

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values

> ....wrap any replaceable values in angle brackets (<>) with the required command parameter, using underscores (_) between words as necessary for legibility.